views/download.handlebars

```diff
@@ -20,8 +20,11 @@
   <div id="download-progress" hidden="true">
     <div id="dl-title" class="title"></div>
     <div class="description" data-l10n-id="downloadingPageMessage"></div>
-    <!-- progress bar here -->
-    <div class="progress-bar" id="dl-progress">
+    <div class="progress-bar">
+      <svg id="progress" width="166" height="166" viewPort="0 0 166 166" version="1.1">
```

Comment:
This could be factored into another template but I'm considering a different template approach in general atm.